mne_qt_browser/_pg_figure.py

```diff
@@ -1862,13 +1861,10 @@ def __init__(self, main):
         self.setLayout(layout)
         self.show(center=False)
 
-    def _chkbx_changed(self, label):
+    def _chkbx_changed(self, checked, label):
```

Comment:
With this signature, calls directly to this method became `self._chkbx_changed(None, labels)` and the call through the Qt signal now works. I was not a big fan of adding this `None` argument everywhere to accomodate the Qt signal case, but I don't see a simpler fix.

Reply:
How about this?
```suggestion
    # For backward compat when "checked" wasn't present, wrap in case of 1 arg
    def _chkbx_changed(self, checked=True, label=None):
        if label is None:
            label = checked
            checked = True
```

Reply:
Done, without setting `checked` to True (anyway the status of the button is not used, so I don't think this is needed).
Tests from `mne-qt-browser` with `mne-main` and tests from `mne-main` with this PR both passes.